### PR TITLE
:sparkles: Allow deletion of personal data

### PR DESCRIPTION
**Pull request by gabriel518**
voir #36 
J'ai ajouté une image par défaut
image
capture d ecran 6

### CODE REVIEW

File: server/scripts/prisma_new_service/index.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const main = async () => {
   }
 
   // Deploy the service
-  await run('prisma deploy', {
+  await run('C:\\Users\\Zenika\\Desktop\\Gabriel\\FAQ\\server\\node_modules\\.bin\\prisma.cmd deploy', {
```
Comment (hgwood) [inline server/scripts/prisma_new_service/index.js]: Oups, je pense que tu as validé ça sans le vouloir :)

**Comment by hgwood** [comment]
Pour aller jusqu'au bout il faudrait aussi effacer le auth0Id. @gabriel518 on en parle quand on se voit.

**Comment by Errorname** [comment]
Il faudrait aussi déconnecter l'utilisateur!

**Comment by hgwood** [comment]
peut-être aussi demander une confirmation du coup ^^'

**Comment by hgwood** [comment]
Tried to make a "are you sure" prompt but could not find a solution. FAQ has prompts already but it looks like they only are triggered by user navigation which is not the case here.

**Comment by Errorname** [comment]
There is currently UI components for prompt. So you can add this simple code:

```js
if (prompt("Are you sure to delete your account?")) {
  // do stuff
}
```

**Comment by hgwood** [comment]
Oh OK it's simply window.prompt. I don't know why I was expecting React to need something specific.

**Comment by hgwood** [comment]
New tooltip and confirmation:

image

What do you think?

**Comment by Errorname** [comment]
I would have prefered the primary button to be "Cancel" instead of "Ok", but since we can't change it, this will do for now. (We will think about build a Prompt UI component later).

One thing we can do: move the "Forget me" button further from the "save" button (to prevent accidental deletion)

**Comment by Errorname** [comment]
Maybe we can put the button in the GDPR card below

**Comment by Errorname** [comment]
I rebased this PR onto master and added a stronger confirmation before deletion. It's inspired from the one Github uses:

screenshot 2019-02-22 at 22 29 43
screenshot 2019-02-22 at 22 29 52
screenshot 2019-02-22 at 22 30 02

File: src/components/Card/Card.jsx [hunk 2]
```diff
@@ -15,4 +19,8 @@ Card.propTypes = {
   className: PropTypes.string
 }
 
+Card.Title = CardTitle
+Card.Text = CardText
+Card.Actions = CardActions
```
Comment (hgwood) [inline src/components/Card/Card.jsx]: 👍

**Pull request merged**